### ISSUE TITLE
getEvolutionChain returning CORS error

### ISSUE THREAD

**Issue by deadbirdi** [anshumanv/evolutions-webapp#13]
##### This issue is about getEvolutionChain returning a [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS) error.
---
When using the `evolution-search` branch of the project, we're receiving this error, which seems to be a CORS issue:

@joshuahunt10 Can you paste the stack trace you're getting?

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by anshumanv** [comment]
Please can someone fix this 😢

**Comment by juffalow** [comment]
If you only want to test it on your local machine, just install some CORS extension. But if you want to use it globally it won't work.
https://en.juffalow.com/other/backbone-js-and-cross-domain-api/

**Comment by anshumanv** [comment]
@juffalow can you fix this ?

**Comment by juffalow** [comment]
No. It has to be "fixed" on the server side. Unless the requested server will not respond with CORS headers, it will not work.

**Comment by anshumanv** [comment]
Ah okay, thanks for the follow up anyways 👍

**Comment by deadbirdi** [comment]
Issue 'fixed' #14. I utilized another library of pokemon for returning evos and prevos.
